### PR TITLE
Changed pipelines to match specification

### PR DESCRIPTION
**Pull request by SG-Noxoreos**
# Pull Request Template

## Description

The Shopgate developer portal specifies the structure of all pipelines and with this change the pipelines `shopgate.cart.updateProducts.v1` and `shopgate.cart.deleteProducts.v1` will meet the specification. The change designed to be compatible with the previous frontend as well.

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ## [Unreleased]
 ### Added
 - localisation for the message a coupon is invalid
+- changed pipelines `shopgate.cart.updateProducts.v1` and `shopgate.cart.deleteProducts.v1` to meet specification in the Shopgate developer portal
```
Comment (augsteyer) [inline CHANGELOG.md:11]: seems like you even have the verbiage `changed`, so it would not go under the `### Added` column :P

**Comment by augsteyer** [comment]
you should ask in channel for reviewers, we sorta did not know about it.
Question about this, which PWA version is ready for this? Do we need a separate QA for it?

**Review by augsteyer — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (SG-Noxoreos) [reply CHANGELOG.md:11]: Okay, misunderstood the title for something else. will add a `### Changed` title

**Comment by augsteyer** [comment]
Would I be correct when saying that these changes are backwards compatible with PWA 5.9.x? So adding them to a minor version is fine, ya?

**Review by augsteyer — APPROVED** [review]

**Pull request merged**